bin/.travis/test.sh

```diff
@@ -78,8 +78,8 @@ printf "\Integration: Behat testing on ez_php:latest and ez_php:latest-dev with
 cd volumes/ezplatform
 
 # Tag image as eZ Platform extends on of these exact images and we don't want it to pull in remote
-docker tag ez_php:latest "ezsystems/php:7.1-${FORMAT_VERSION}"
-docker tag ez_php:latest "ezsystems/php:7.0-${FORMAT_VERSION}"
+docker tag ez_php:latest "ezsystems/php:7.2-${FORMAT_VERSION}"
+docker tag ez_php:latest "ezsystems/php:7.1-v1"
```

Comment:
Why not {FORMAT_VERSION} here?

Reply:
Once we change to v2 the aliasing will fail as what platform uses right now is `ezsystems/php:7.1-v1`

I now realize 1.7 is still using `ezsystems/php:7.0-v1`, so I should actually not remove the 7.0 alias here.